app/views/recipes/index.html.haml

```diff
@@ -0,0 +1,2 @@
+%h1 FoodHub
+= link_to 'Sign up',  new_user_registration_path
```

Comment:
You probably want this link to be visible on all pages throughout the application. Why not place it in the main layout template?